python/cuml/test/test_umap.py

```diff
@@ -230,7 +230,7 @@ def test_umap_fit_transform_trust(name, target_metric):
         data = wine.data
         labels = wine.target
     else:
-        data, labels = make_blobs(n_samples=5000, n_features=10,
+        data, labels = make_blobs(n_samples=500, n_features=10,
```

Comment:
Just had one quick question, have you seen any issues of reducing the data so much? i.e. it would be nice to run the test a number of times to be sure no flakiness is being introduced

Reply:
good point

Reply:
I saw no failures with 100 retries of this test.